### PR TITLE
Fix problems found by codespell

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: pkg/mount/mount.go [hunk 1]
```diff
@@ -90,7 +90,7 @@ func RecursiveUnmount(target string) error {
 		if err := Unmount(m.Mountpoint); err != nil && i == len(mounts)-1 {
 			return err
 			// Ignore errors for submounts and continue trying to unmount others
-			// The final unmount should fail if there ane any submounts remaining
+			// The final unmount should fail if there and any submounts remaining
```
Comment (TomSweeneyRedHat) [inline pkg/mount/mount.go:93]:
```suggestion
			// The final unmount should fail if there are any submounts remaining
```

**Comment by TomSweeneyRedHat** [comment]
One nit and it looks like you need a rebase already.

**Review by TomSweeneyRedHat — COMMENTED** [review]
LGTM

**Comment by rhatdan** [comment]
@vrothberg @saschagrunert @giuseppe @ashley-cui @QiWang19 @mheon PTAL

**Review by ashley-cui — COMMENTED** [review]
LGTM

**Pull request merged**